app/helpers/alchemy/admin/pages_helper.rb

```diff
@@ -51,6 +51,16 @@ def page_layout_missing_warning
           class: 'inline warning icon'
         )
       end
+
+      # Renders a warning icon with a hint
+      # that explains the user that their alchemy role does not
+      # allow them to make edits to this page
+      def page_roles_denial_warning
```

Comment:
I don't see any reason to introduce a new helper only for the one usage. It's ok to have this in the view